api/agent/iofs.go

```diff
@@ -64,6 +65,13 @@ func newDirectoryIOFS(ctx context.Context, cfg *Config) (*directoryIOFS, error)
 		}
 		return nil, fmt.Errorf("cannot create tmpdir for iofs: %v", err)
 	}
+	if !cfg.DockerDisableSecurity {
+		err = os.Chown(iofsAgentDir, docker.FnUserId, docker.FnGroupId)
```

Comment:
pedantic: not sure if it would be better to do at create time (reduce syscall load, latency, etc) -- fully realize the horror that is the iofs stuff, it's maybe not worth it.

Reply:
we use `ioutil.TempDir` to create the dir, which doesn't allow to set user, but yeah, I can add similar function